docs/app/Examples/views/Statistic/Types/StatisticStatisticsExample.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import {Statistic} from 'stardust';
+const {Statistics, Label, Value} = Statistic;
```

Comment:
We have spec'd to restructure to sub-components as static properties on the parent component, see #99.  Up until now this has only included component parts, or elements internal to the component.  For instance, a `<Menu.Item />` for a `<Menu />`.

This PR proposes (shown on ln 3) that we also include the plural "group" element as a sub-component.

**Pros:**
- Only a single property (parent component) needs to be exposed in the Stardust module.
- Only a single import is required to use the full spectrum of markup for a given component. Opposed to importing the component and it's plural name for groups, as it is currently.
- I think it is the proper cohesion as you should never use `<Items />` unless you are also using `<Item />`.

**Cons:**
- The naming is a little odd and ugly (note I destructured the sub-components above).  This is especially true with longer component names and without destructuring:

  ```jsx
<Statistic.Statistics>
  <Statistic>
    <Statistic.Value>22</Statistic.Value>
    <Statistic.Label>Faves</Statistic.Label>
  </Statistic>
  <Statistic>
    <Statistic.Value>31,200</Statistic.Value>
    <Statistic.Label>Views</Statistic.Label>
  </Statistic>
</Statistic.Statistics>
  ```

**Ideas:**
- We could use the sub-component name `Group` for the plural names.  It would give us a standard across all components and slightly clearer syntax (below). However, it does break the alignment with JS class names and HTML class names (`className='statistics'` and `class Group extends Component {...`)

  ```jsx
<Statistic.Group>
  <Statistic>
    <Statistic.Value>22</Statistic.Value>
    <Statistic.Label>Faves</Statistic.Label>
  </Statistic>
  <Statistic>
    <Statistic.Value>31,200</Statistic.Value>
    <Statistic.Label>Views</Statistic.Label>
  </Statistic>
</Statistic.Group>
  ```

Feedback?